src/lib/providers/parser.ts

```
@@ -999,3 +999,206 @@ function transformTraktRecommendations(
+        case 1:
+            return "Female";
+        case 2:
+            return "Male";
+        case 3:
+            return "Non-binary";
+        default:
+            return null;
+    }
+}
+
+export function parsePersonDetails(personData: any): PersonDetails {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)